packages/ag-charts-community/src/chart/series/polar/donutSeries.ts

```diff
@@ -310,7 +310,18 @@ export class DonutSeries extends PolarSeries<DonutNodeDatum, Sector> {
 
     async createNodeData() {
         const { id: seriesId, processedData, dataModel, angleScale } = this;
-        const { rotation } = this.properties;
+        const { rotation, innerRadiusRatio, innerRadiusOffset } = this.properties;
+
+        if (innerRadiusRatio == null && innerRadiusOffset == null) {
+            Logger.warnOnce(
+                'Either an [innerRadiusRatio] or an [innerRadiusOffset] must be set to render a donut series.'
+            );
+
+            this.zerosumOuterRing.visible = true;
+            this.zerosumInnerRing.visible = true;
```

Comment:
It might be better to add some flags to the result of this method, and apply this visibility change in a later update phase (ideally we wouldn't have side-effects in `processData()` and `createNodeData()` calls, defering updating the scene graph to later in the processing flow in the `update()` call).